Documentation clean up (#299)
Bandwidth/api-docs: site/docs/voice/about.mdx

`https://voice.bandwidth.com/api/v2/accounts/{accountId}`

**Note** HTTP requests to our API use HTTP Basic authentication, which refers to your account's username and password.

## Calls

For help on how to make the proper requests to the API, check out the [API Reference](/apis/voice#operation/createCall) section of the docs. To quickly start making and receiving calls, check out the [guide](/docs/voice/quickStart).

### Creation/Acceptance

> A <Highlight color="#079CEE">POST</Highlight> to `/calls` creates a call object in Bandwidth's network and returns a promise in the form of a unique `callId`. When creating a call you provide an `answerUrl` in the request body, and upon answer Bandwidth will send a webhook to the provided URL confirming and expecting BXML in the HTTP response to continue with the call.

Reviewer: `promise`? That seems like odd phrasing to me since we're not talking about JavaScript.

Reviewer: ```suggestion
A <Highlight color="#079CEE">POST</Highlight> to `/calls` creates a call object in Bandwidth's network and returns a promise in the form of a unique `callId`. When creating a call you provide an `answerUrl` in the request body, and upon the call being answered, Bandwidth will send a webhook to the provided URL to inform your application and retrieve BXML from it in order to continue with the call.
```

To handle inbound calls to your Bandwidth numbers - you associate a [voice application](/docs/account/setup#create-a-voice-application) to a group of numbers, known as a Location or Sip-Peer. When incoming calls are received to any of those numbers, Bandwidth sends an [`initiate`](/docs/voice/webhooks/initiate) event webhook to the URL defined in your application and expects BXML in the HTTP response to continue with the call.

### Call Directions

> The direction of which the call is coming from may determine how and where callbacks are sent, and whether they may submit more BXML. There are two directions, inbound and outbound. Inbound calls refer to calls received by your application, and outbound calls are calls that originate from your application.

Reviewer: ```suggestion
The direction of which the call is coming from may determine how/where callbacks are sent and whether they may submit more BXML. The direction of a call is either inbound or outbound. Inbound calls refer to calls received by your application, and outbound calls are calls that originate from your application (via API requests or BXML verbs).
```

### Webhooks

> The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certain webhooks expect BXML in the HTTP response to keep the call alive, while others are simply informative. These are referred to as callbacks in the previous docs.

Reviewer: ```suggestion
The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certain webhooks expect BXML in the HTTP response to keep the call alive, while others are simply informative. These used to be referred to as "callbacks".
```

### STIR/SHAKEN

The Bandwidth API uses STIR/SHAKEN to help combat fraudulent calls and phone number spoofing. This info can be seen in the [`initiate`](/docs/voice/webhooks/initiate) webhook. For more information, look [here](https://www.bandwidth.com/regulations/stir-shaken/) for Bandwidth's explanation of what it is and why it's important.

## Features and Services of Note

### Text-to-Speech

> Use our [SpeakSentence](/docs/voice/bxml/speakSentence) verb to convert typed words to speech, powered by Amazon Polly. Seventeen voices, twenty-six languages. Use [SSML](https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/) to customize your audio to include pauses, emphasis, and much more.

Reviewer: ```suggestion
Use our [SpeakSentence](/docs/voice/bxml/speakSentence) verb to convert typed words to audible speech on the call. Powered by Amazon Polly, which allows us to support seventeen voices and twenty-six languages. Use [SSML](https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/) to customize your audio to include pauses, emphasis, and much more.
```

### Recording

> With the [Record](/docs/voice/bxml/record) verb, customers can save calls to play back later.

Reviewer: I'm not personally a fan of referring to our customers as `customers` here, since these are external docs. I'd prefer `you`, but whatever we use, we should be consistent across our (voice API) docs.

Reviewer: ```suggestion
With the [Record](/docs/voice/bxml/record) verb, you can save calls to play back or analyze later.
```

Author: I was using 'you' before, but figured that was weirdly informal. Not sure about consistency, but I'll switch it.

### Transcription

Convert your calls to text using the [Record](/docs/voice/bxml/record) verb's `transcribe` function.

### Conferencing

> The [Conference](/docs/voice/bxml/conference) BXML Verb allows customers to create conference calls, including a [coach](/docs/voice/bxml/conference#join-a-conference-as-a-coach) function to have a user only heard by select attendees of the conference.

Reviewer: ```suggestion
The [Conference](/docs/voice/bxml/conference) BXML Verb allows you to create conference calls—including with the [coach](/docs/voice/bxml/conference#join-a-conference-as-a-coach) feature that allows for a call to be selectively bridged with certain calls within a conference.
```

### Answering Machine Detection

> [Answering Machine Detection](/docs/voice/guides/machineDetection) allows customers to distinguish whether a human or machine is speaking on a call.

Reviewer: ```suggestion
[Answering Machine Detection](/docs/voice/guides/machineDetection) allows you to determine whether a human or a machine has answered a call.
```

### Caller Name Service (CNAM)

> Bandwidth's [Caller Name service](/docs/voice/guides/cnamPerDip) provides the business name on VoIP calls in the US and Canada using our CNAM database. This feature must be enabled.

Reviewer: ```suggestion
Bandwidth's [Caller Name service](/docs/voice/guides/cnamPerDip) provides the business name on VoIP calls in the US and Canada using our CNAM database. This feature is not enabled by default.
```

### Recordings

Bandwidth will keep recordings for up to 30 days. After 30 days the recordings will be deleted from Bandwidth's servers.

### Transcriptions

> Bandwidth will keep transcriptions for up to 30 days. After 30 days the transcriptions will be deleted from Bandwidth's servers.

Reviewer: ```suggestion
Bandwidth will keep transcriptions for up to 30 days. After 30 days, the transcriptions will be marked for deletion, which may take up to `48` hours to fully process.
```

## Supported Languages and Environments

> Supported languages for call control include XML, Java, C#, Ruby, NodeJs, Python, and PHP. Examples of the Bandwidth library in these languages are provided throughout the docs.

Reviewer: `XML` here just meant BXML, right?
```suggestion
SDKs are provided for Java, C#, Ruby, NodeJs, Python, and PHP. Examples of the Bandwidth library in these languages are provided throughout the docs.
```
